helm-chart/jupyterhub-ssh/templates/secret.yaml

```diff
@@ -10,3 +10,4 @@ stringData:
     {{- .Values | toYaml | nindent 4 }}
   hostKey: {{ .Values.hostKey | required "hostKey must be set to a valid SSH Private key" | quote }}
   hubUrl: {{ .Values.hubUrl | required "hubUrl must be set to a valid JupyterHub URL" | quote }}
+  sshConnTimeout: {{ .Values.ssh.timeout | default 30 | quote }}
```

Comment:
The other configurations are exposed here because hostKey and hubUrl needs to be read directly from a k8s resource manifests referencing a specific key and value from this k8s Secret resource. But, when we use the timeout configuration, we do it from Python where we can load the entire values.yaml file where this configuration is available as well already.

In short, we should only have entries here if needed as a direct reference to its value from another k8s manifest.

```suggestion
```